rosdep/base.yaml

```
@@ -495,6 +495,15 @@ catch2:
   ubuntu:
     '*': [catch2]
     focal: null
+cbindgen:
+  alpine: [cbindgen]
+  arch: [cbindgen]
+  debian: [cbindgen]
+  fedora: [cbindgen]
+  rhel: [cbindgen]
```

CI says that this package is not available for RHEL 8, so this should probably be:

```suggestion
  rhel:
    '*': [cbindgen]
    '8': null
```